tensorboard/webapp/widgets/histogram/histogram_v2_component.ts

```diff
@@ -130,6 +130,7 @@ export class HistogramV2Component
     },
   };
   private domInitialized = false;
+  private domVisible = false;
 
   constructor(private readonly changeDetector: ChangeDetectorRef) {
     // `data` may not be available at the constructor time. Since we recalculate
```

Comment:
Now that we no longer have `ngAfterViewInit`, let's update this comment that refers to it.